index.html

```diff
@@ -29,7 +36,16 @@
       <!---------------- Mobile Menu ------------>
       <header class="pf-header">
         <a class="pf-logo" href=""><img class="pf-logo" src="./img/F_logo.png" alt="Fahim logo" /></a>
-        <nav class="material-symbols-outlined">menu</nav>
+        <a href="#" class="" id="toggleMenuButton"><i class="fa fa-bars" id="m-icon" aria-hidden="true"></i></a>
+
+        <div hidden class="menu-show" id="toggleMenuItems">
+          <a href="#"><i class="fa fa-times" id="x-icon" aria-hidden="true"></i></a>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ behave as buttons when the mouse hovers over them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events.

Reply:
Thanks a lot for the suggestion. I will surely do that.